lang/java/avro/src/main/java/org/apache/avro/file/DataFileStream.java

```diff
@@ -246,25 +266,29 @@ public ByteBuffer nextBlock() throws IOException {
   }
 
   /** Expert: Return the count of items in the current block. */
-  public long getBlockCount() { return blockCount; }
+  public long getBlockCount() {
+    return blockCount;
+  }
 
   /** Expert: Return the size in bytes (uncompressed) of the current block. */
-  public long getBlockSize() { return blockSize; }
+  public long getBlockSize() {
+    return blockSize;
+  }
 
   protected void blockFinished() throws IOException {
     // nothing for the stream impl
   }
 
   boolean hasNextBlock() {
     try {
-      if (availableBlock) return true;
-      if (vin.isEnd()) return false;
-      blockRemaining = vin.readLong();      // read block count
-      blockSize = vin.readLong();           // read block size
-      if (blockSize > Integer.MAX_VALUE ||
-          blockSize < 0) {
-        throw new IOException("Block size invalid or too large for this " +
-          "implementation: " + blockSize);
+      if (availableBlock)
+        return true;
+      if (vin.isEnd())
```

Comment:
it should probably add braces here, if we don't allow single line if's then we should require braces, IMO.  It is a lot more error prone when not on one line.